The main webkit library links against stdc++ so we need to use g++ for 
linking. The easiest way to achieve this is to add a dummy cpp file.
This also means we have to add AC_PROG_CXX else libtool 2.2.2 gets upset.

RP - 14/4/08 

Upstream-Status: Inappropriate [configuration]

--- webkit.orig/src/Makefile.am
+++ webkit/src/Makefile.am
@@ -1,11 +1,11 @@
 AM_CPPFLAGS=-DPKGDATADIR=\"$(pkgdatadir)\"
 AM_CFLAGS = -Wall -export-dynamic $(WEB_CFLAGS)
 
 bin_PROGRAMS = web2
 
-web2_SOURCES = web2.h web2-main.c web2-utils.c web2-utils.h
+web2_SOURCES = web2.h web2-main.c web2-utils.c web2-utils.h dummy.cpp
 web2_LDADD   = $(WEB_LIBS)
 
 MAINTAINERCLEANFILES = config.h.in Makefile.in
 
 uidir = $(pkgdatadir)
Index: webkit/configure.ac
===================================================================
--- webkit.orig/configure.ac	2008-04-14 22:15:25.000000000 +0100
+++ webkit/configure.ac	2008-04-14 22:16:31.000000000 +0100
@@ -7,6 +7,7 @@
 
 AC_ISC_POSIX
 AC_PROG_CC
+AC_PROG_CXX
 AC_STDC_HEADERS
 AC_PROG_LIBTOOL
 
